cmd/minishift/cmd/start.go

```diff
@@ -194,6 +194,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	autoMountHostFolders(hostVm.Driver)
+	addProfileToConfig()
```

Comment:
I think this happens quite late in the start of the VM. This means the profile is only added, after the start and automount has happened? So any exit before will prevent the profile from being added?

Reply:
Right. I think it can be before auto mount for sure. But do we want to do it before VM start, I am choose to do it after the start as so many things can go wrong during start.

Reply:
When did the entry for the previous minishift instance exist? I believe this happened at the moment the docker machine was triggered? I believe doing this ASAP. if during the start it goes wrong, (like no network), are we able to delete now?